src/main.py

```diff
@@ -25,71 +25,89 @@
 
 @app.route("/")
 def index():
-    return render_template("index.html", the_title="Paint Drying")
+	return render_template("index.html", the_title="Paint Drying")
+
+
+@app.route("/login", methods=['POST', 'GET'])
+def login():
+	global current_user_email
+
+	form = LoginForm()
+	if form.validate_on_submit():
+		users = db.get_clients(lambda client: client["email"] == form.email.data)
+		if not users:
+			raise Exception('User with given email does not exist')
```

Comment:
we shouldn't raise exceptions in critical path, instead we should return 4xx error code